bin/oss/index.js

```diff
@@ -96,14 +96,59 @@ class OSS extends Command {
     async apply3rdPartiesLicenses() {
         let stdout;
         try {
-            stdout = (await execute('yarn', ['licenses-csv'], ROOT)).stdout;
+            stdout = (await execute('yarn', ['licenses', 'list', '-R', '--json'], ROOT)).stdout;
         } catch (e) {
             // eslint-disable-next-line no-console
             console.log(e);
         }
 
-        if (stdout) {
-            return fs.writeFile(path.join(ROOT, 'LICENSES-3rdparty.csv'), stdout);
+        const licenses = new Map();
+
+        const nameRegex = /^(@[.*?]\/.*?|[^@]+)@(.+?):.+?$/;
```

Comment:
Can you add a comment explaining what it does with some examples please.

Reply:
I added a comment, please tell me what you think about it 👌